### PR TITLE
RD-5713: Updated popups components

### PR DESCRIPTION
**Pull request by Vorbert-Kruk**
## Description

Changes consists of
- fully migrating components residing within `src/components/popups` directory
- migrating away from implicit lodash imports, in order of using explicit ones
- removing redundant `@ts-expect-error` comments 

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

N/A

## Documentation

N/A

### CODE REVIEW

File: src/components/popups/PopupConfirm/PopupConfirm.stories.tsx [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 };
 
-// @ts-expect-error TS(2739) FIXME: Type '{ trigger: Element; content: string; }' is m... Remove this comment to see the full error message
-export const basic = () => <PopupConfirm trigger={<button type="button">Delete</button>} content="Delete this file?" />;
+export const basic: Story<Required<PopupConfirmProps>> = () => (
+    <PopupConfirm trigger={<button type="button">Delete</button>} content="Delete this file?" />
+);
 basic.storyName = 'Default';
 
 export const noCancelButton = () => (
```
[the remark this answers is not in the document]
Comment (Vorbert-Kruk) [reply src/components/popups/PopupConfirm/PopupConfirm.stories.tsx:19]: Sure, I don't know why haven't I thought about it earlier 😅 
It has been done not only to the `PopupConfirm.stories.tsx` but also to the other ones aswell 🎸

File: src/components/popups/PopupConfirm/PopupConfirm.tsx [hunk 2]
```diff
@@ -1,15 +1,45 @@
...
 
 /**
  * PopupConfirm is a component which uses [Popup](https://react.semantic-ui.com/modules/popup) component to display
  * popup with content enhanced by two buttons - OK and Cancel.
  *
  * All props supported by the underlaying `Popup` component are passed down to it.
  */
-// @ts-expect-error TS(7031) FIXME: Binding element 'content' implicitly has an 'any' ... Remove this comment to see the full error message
-export default function PopupConfirm({ content, onCancel, onCanConfirm, onConfirm, defaultOpen, ...popupProps }) {
+export default function PopupConfirm({
```
[the remark this answers is not in the document]
Comment (Vorbert-Kruk) [reply src/components/popups/PopupConfirm/PopupConfirm.tsx:35]: As the `FunctionComponent` type is coming with `children` prop, I think that we could omit it
WDYT? 🤔

[the remark this answers is not in the document]
Comment (qooban) [reply src/components/popups/PopupConfirm/PopupConfirm.tsx:35]: OK, for the time being it's fine as long as the types (function arguments = props & return type = JSX.Element?) are proper.

In React 18 they changed `React.FunctionComponent` type and it no longer contain `children` prop. Ref. https://solverfox.dev/writing/no-implicit-children/

**Review by qooban — APPROVED** [review]

File: src/components/popups/PopupConfirm/PopupConfirm.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vorbert-Kruk) [reply src/components/popups/PopupConfirm/PopupConfirm.tsx:35]: > OK, for the time being it's fine as long as the types (function arguments = props & return type = JSX.Element?) are proper.   

Agreed

> In React 18 they changed React.FunctionComponent type and it no longer contain children prop. Ref. https://solverfox.dev/writing/no-implicit-children/

Oh, nice!
I'll try to remember that when we'd be upgrading react 💪

**Pull request merged**